src/services/wasm/index.ts

```diff
@@ -61,7 +61,7 @@ export const TranslateToLatest = (lineage: string, input: string): void => {
 
 export const TranslateToVersion = (lineage: string, input: string, version: string): void => {
   // @ts-ignore
-  const res = translateToLatest(lineage, input);
+  const res = translateToVersion(lineage, input, version);
```

Comment:
Not related to this PR, but we'll need to update these function names to start with a lower case, the upper case is conventional for React components or JS Classes, which can be confusing. I understand it will clash with the exported names from the WASM, so we might need to slightly rename them.

Reply:
Good point! I'll be happy with any suggestion, honestly!
Perhaps adding some prefix to the WASM ones, or smth like that.
Additionally, I also think that it could be good idea to explore any way to type hint those, maybe 🤔